src-v2/Not/Not.Application/Ports/CRUD/ISetRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Not.Domain;
+
+namespace Not.Application.Ports.CRUD;
+
+public interface ISetRepository<T> : ICreate<T>, IReadAll<T>, IUpdate<T>, IDelete<T>
```

Comment:
Good job here. This shows a level of understanding of composition of our `IRepository` abstractions.